weebnation.py

```diff
@@ -58,5 +65,10 @@ async def findAnime(self,message):
             else:
                 msg = "😁: Ich habe folgende Anime gefunden:\n"
                 for i in range(len(result)):
-                    msg = msg + ("%d. %s (%s) [%s]\n" % (i+1, result[i][0],result[i][1],result[i][2]))
-                await message.channel.send(msg)
\ No newline at end of file
+                    msg = msg + result[i][0] + " :link: " + result[i][1] + "\n"
+                    print(i)
+                    if i+2 > 3:
+                        break
+            else:
```

Comment:
else without if

Reply:
what, in my push was no lonely else...
there is an
**if** ...stuff...
**elif** ...more stuff...
**else** ...end stuff...